src/test/java/org/killbill/billing/plugin/vertex/VertexTaxCalculatorTest.java

```diff
@@ -176,4 +177,40 @@ public void testComputeDryRun() throws Exception {
                 .addResponse(any(UUID.class), any(UUID.class), anyMap(), any(ApiSuccessResponseTransactionResponseType.class), any(DateTime.class), any(UUID.class));
         verify(vertexApiClient).calculateTaxes(argThat(arg -> SaleMessageTypeEnum.QUOTATION.equals(arg.getSaleMessageType())));
     }
+
+    @Test(groups = "fast")
+    public void testTaxDescriptionLogic() throws Exception {
+        //given
+        final boolean isDryRun = true;
```

Comment:
nit, I think `isDryRun` should be false for this test as we are not specifically testing dryRun.